model-optimizer/mo/main.py

```diff
@@ -144,20 +144,18 @@ def prepare_ir(argv: argparse.Namespace):
 
     # This try-except is additional reinsurance that the IE
     # dependency search does not break the MO pipeline
-    try:
-        argv.ie_is_available = find_ie_version(silent=argv.silent)
-
-        if not argv.ie_is_available and not argv.silent:
-            print("[ WARNING ] Could not find the Inference Engine Python API. At this moment, the Inference Engine dependency is not required, but will be required in future releases.")
-            print("[ WARNING ] Consider building the Inference Engine Python API from sources or try to install OpenVINO (TM) Toolkit using \"install_prerequisites.{}\"".format(
+    def raise_ie_not_found():
```

Comment:
Did we analyse all distribution channels?
I think we shouldn't have problems if user uses MO from the default package.

But if I as a developer try to run MO from the source tree after this PR I will get the error that nGraph and IE Python APIs weren't found.
Can we add a new CMake target (or something like that) for the MO, in order to I will be able to run this target and it will automatically prepare all necessary components and environment for the MO?

Reply:
I added cmake custom target with `model_optimizer` name which gather all necessary dependencies to make MO work. In case if some of required targets are not available then `model_optimizer` target will be disabled.